examples/utils/get-tinylicious-container/src/test/insecureTinyliciousUrlResolverTest.spec.ts

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+// import { DriverHeader, IFluidResolvedUrl } from "@fluidframework/driver-definitions";
+import { ensureFluidResolvedUrl } from "@fluidframework/driver-utils";
+// import { IUser } from "@fluidframework/protocol-definitions";
+import { IRequest } from "@fluidframework/core-interfaces";
+import { InsecureTinyliciousUrlResolver } from "../getTinyliciousContainer";
+
+describe("Insecure Url Resolver Test", () => {
+    const documentId = "fileName";
+    const hostUrl = "fluid://localhost:3000";
+    let resolver: InsecureTinyliciousUrlResolver;
+
+    beforeEach(() => {
+        resolver = new InsecureTinyliciousUrlResolver();
+    });
+
+    it("Test RequestUrl for url with only document id", async () => {
+        const testRequest: IRequest = {
+            url: `${documentId}`,
+            headers: {},
+        };
+
+        const resolvedUrl = await resolver.resolve(testRequest);
+        ensureFluidResolvedUrl(resolvedUrl);
+
+        const expectedResolvedUrl = `${hostUrl}/tinylicious/${documentId}`;
+        assert.strictEqual(resolvedUrl.url, expectedResolvedUrl, "resolved url is wrong");
+    });
+
+    it("Test RequestUrl for url with data object ids", async () => {
+        const path = "dataObject1/dataObject2";
+        const testRequest: IRequest = {
+            url: `${documentId}/${path}`,
+            headers: {},
```

Comment:
Don't add headers if its going to be empty. I guess its fine to add in one of the test cases to make sure it doesn't break.
You can probably add a header for "createNew" and make sure things work as expected - In this resolver, the header will just be ignored.